src/app/(pages)/properties/details/[id]/subproperties/[action]/[idSubproperty]/page.js

```diff
@@ -0,0 +1,174 @@
+"use client"
+import Button from "@/components/Form/Button"
+import CustomerForm from "@/components/Form/CustomerForm";
+import StepperHorizontal from "@/components/Steppers/StepperHorizontal";
+import Link from "next/link"
+import { useEffect, useState } from "react";
+import {  toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { MdAccountBox,MdShoppingCart} from "react-icons/md";
+import { useSelector } from "react-redux";
+import { useDispatch } from "react-redux";
```

Comment:
please avoid the double import and use {useSelector, useDispatch} from 'react-redux'